fem/bilinearform.cpp

```diff
@@ -479,6 +488,43 @@ void BilinearForm::Assemble(int skip_zeros)
             }
          }
       }
+
+      // Patch-wise integration
+      if (mesh->NURBSext)
```

Comment:
Similar to above, it's probably better to check for NURBS FE space:
```suggestion
      if (fes->GetNURBSext())
```